utils/tintin/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tintin
+PKG_VERSION:=2.02.20
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/scandum/tintin/releases/download/$(PKG_VERSION)/
+PKG_HASH:=331673e6ee3c945cf27e1c0d71cec1225c9d992588ed73b2a707c4c49523e8d2
+
+PKG_MAINTAINER:=scandum <ivdhoven@gmail.com>
```

Comment:
Who is this? That's not you. You copy&pasted it from somewhere, right? Does he agreed that he is going to be maintainer of this package in this repository?

Reply:
I'm going to be maintainer of this package in this repository. I haven't figured out what PKG_MAINTAINER means before